zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -671,18 +729,18 @@ private[consumer] object Runloop {
                   pollFrequency,
                   pollTimeout,
                   requestQueue,
-                  commitQueue,
+                  commandQueue,
                   lastRebalanceEvent,
                   partitions,
                   rebalancingRef,
                   diagnostics,
                   shutdownRef,
                   offsetRetrieval,
                   userRebalanceListener,
-                  subscribedRef,
                   restartStreamsOnRebalancing,
                   currentStateRef
                 )
+      _ <- ZIO.addFinalizer(ZIO.logInfo("Shut down Runloop"))
```

Comment:
🌕 I don't think we should log this in out users logs at the INFO level. IMO, they don't care and they shouldn't have to care.

```suggestion
      _ <- ZIO.addFinalizer(ZIO.logDebug("Shut down Runloop"))
```